src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Better PWA icons with maskable icons
+subhead: A new icon format to use adaptive icons on Android.
+description: |
+  A description of the article that will appear in search results.
+authors:
+  - tigeroakes
+date: 2019-12-09
+hero: hero.jpg
+# You can adjust the fit of your hero image with this property.
+# Values: contain | cover (default)
+# hero_fit: contain
+
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# thumbnail: thumbnail.jpg
+
+alt: A description of the hero image for screen reader users.
+
+# You can provide a custom thumbnail and description for social media cards.
+# Images should be 1200 x 630.
+# If no social thumbnail is provided then the post will attempt to fallback to
+# the post's thumbnail or hero from above. It will also reuse the alt.
+# social:
+#   google:
+#     title: A title for Google search card.
+#     description: A description for Google search card.
+#     thumbnail: google_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   facebook:
+#     title: A title for Facebook card.
+#     description: A description for Facebook card.
+#     thumbnail: facebook_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   twitter:
+#     title: A title for Twitter card.
+#     description: A description for Twitter card.
+#     thumbnail: twitter_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+If you've installed a Progressive Web App on a recent Android phone, you might notice the icon shows up with a white background. {{image}}. Android Oreo introduced adaptive icons, which display app icons in a variety of shapes across different device models. Icons that don't follow this new format are given white backgrounds.
+
+Maskable icons are a new icon format that let your Progressive Web App use adaptive icons. If you supply a maskable icon, your icon can fill up the entire shape and look great on all Android devices. Firefox and Chrome have recently added support for this new format, and you can adopt it in your apps.
+
+## Are my current icons ready?
+
+Since maskable icons need to support a variety of shapes, you supply an opaque image with some padding that the browser can later crop into the correct shape and size. The important parts of your icon, such as your logo, should be within the middle 80% of the icon. The outer 10% edge may be cropped.
```

Comment:
```suggestion
Since maskable icons need to support a variety of shapes, you supply an opaque image with some padding that the browser can later crop into the desired shape and size. It's best not to rely on any particular shape, since it can vary by browser and per platform.

Luckily, there's a well-defined and [standardized](https://w3c.github.io/manifest/#icon-masks) "minimum safe zone" that all browsers respect. The important parts of your icon, such as your logo, should be within the middle 80% of the icon. The outer 10% edge may be cropped.
```